### PR TITLE
Prepare the skeleton for PageInfo endpoints

### PR DESCRIPTION
**Pull request by ryanpbrewster**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/common/Implicits.scala [hunk 1]
```diff
@@ -113,6 +113,9 @@ final class MapExtensionOps[A, B](xs: Map[A, B]) {
   def mapValuesStrict[C](fn: B => C): Map[A, C] = xs.map { case (k, v) => k -> fn(v) }
   def flatMapValues[C](fn: B => Option[C]): Map[A, C] = xs.flatMap { case (k, v) => fn(v).map(k -> _) }
   def filterValues(predicate: B => Boolean): Map[A, B] = xs.filter { case (k, v) => predicate(v) }
+  def traverseByKey(implicit ord: Ordering[A]): Seq[B] = {
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/Implicits.scala:116]: Why not just on `Seq`s then?

Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/common/Implicits.scala:116]: What does this mean?

Comment (ryanpbrewster) [inline kifi-backend/modules/common/app/com/keepit/common/Implicits.scala:116]: I found that a common pattern in my own code is to call a method

```
val modelById = getByIds(ids: Set[Id[T]]): Map[Id[T], T]
```

and then extract the values in some order, like

```
modelById.values.sortBy(_.id.get)
```

Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/Implicits.scala:116]: Ah. I forget if Scala lets you provide an implicit for Seq[(A,B)]... I _think_ you can.

**Pull request merged**